docs/pages/management/dynamic-resources/access-list.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: Creating Access Lists with IaC
+description: Use Infrastructure-as-Code tooling to create Teleport AccessLists.
+---
+
+Access Lists allow Teleport users to be granted long term access to resources
+managed within Teleport. With Access Lists, administrators and access list
+owners can regularly audit and control membership to specific roles and
+traits, which then tie easily back into Teleport's existing RBAC system.
```

Comment:
```suggestion
managed within Teleport. With Access Lists, administrators can regularly audit and control membership to specific roles and
traits, which then tie easily back into Teleport's existing RBAC system.
```
Seemed a bit awkward to repeat "access list" twice with different capitalization, and I'm not sure it's clear this point what the difference between an admin and an Access List owner is.